### PR TITLE
Fix Bug 1377464 - Make changes for Mozilla Japan transition

### PR DESCRIPTION
**Pull request by kyoshino**
## Description

Mozilla Japan, an affiliate of Mozilla Foundation, has become independent from Mozilla, so some changes are needed, including removals of redirects and Japan space page.

## Bugzilla link

[Bug 1377464](https://bugzilla.mozilla.org/show_bug.cgi?id=1377464)

## Testing

* Visit `/ja/` and `/ja/about/` to make sure you no longer get redirected to mozilla.jp.
* `/contact/spaces/tokyo/` no longer exists.

## Checklist
- [ ] Requires l10n changes.
- [x] Related functional & integration tests passing.

### CODE REVIEW

File: bedrock/mozorg/redirects.py [hunk 1]
```diff
@@ -271,14 +258,6 @@ def decider(request, **kwargs):
     # bug 957664
     redirect(r'^press/awards(?:/|\.html)?$', 'https://blog.mozilla.org/press/awards/'),
 
-    # bug 987059, 1050149, 1072170, 1208358
-    redirect(r'^ja/about/?$', 'https://www.mozilla.jp/about/mozilla/'),
-    redirect(r'^ja/about/japan/?$', 'https://www.mozilla.jp/about/japan/'),
```
Comment (alexgibson) [inline bedrock/mozorg/redirects.py]: Are we ok that `/ja/about/japan/` will now 404? I'm not entirely clear from [bug 1208358](https://bugzilla.mozilla.org/show_bug.cgi?id=1208358) on the history for why this redirect existed.

File: bedrock/mozorg/urls.py [hunk 2]
```diff
@@ -89,7 +89,6 @@
     page('contact/spaces/portland', 'mozorg/contact/spaces/portland.html'),
     page('contact/spaces/san-francisco', 'mozorg/contact/spaces/san-francisco.html'),
     page('contact/spaces/taipei', 'mozorg/contact/spaces/taipei.html'),
-    page('contact/spaces/tokyo', 'mozorg/contact/spaces/tokyo.html'),
```
Comment (alexgibson) [inline bedrock/mozorg/urls.py]: Is it worth adding a redirect to the `/contact/spaces/` landing page, so this doesn't 404?

Also, should this definitely be removed from the spaces list? Has someone specifically asked for this to happen? It's not quite clear from Mitchell's email what the implications are for the Mozilla Japan space.

**Comment by alexgibson** [comment]
@kyoshino any updates on this one?

**Comment by kyoshino** [comment]
Sorry for the late update 🙇  Talked with the Japan team, and looks like they have to tell a MoCo counterpart about this so they can confirm the change. Putting the **Do Not Merge** label before moving forward.

**Comment by kyoshino** [comment]
I'm closing this PR; the redirects have already been removed in #5005, and the Tokyo space page will be removed as part of [Bug 1393622](https://bugzilla.mozilla.org/show_bug.cgi?id=1393622).

**Pull request closed without merging**